### PR TITLE
Update connect-your-database.md

### PR DESCRIPTION
**Pull request by will-sargent-dbtlabs**
## Description & motivation
Public facing docs do not include the steps necessary to prepare a bastion host for the SSH connection on the AWS side.

### CODE REVIEW

File: website/docs/docs/get-started/connect-your-database.md [hunk 1]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-ssh-tunnel.png" title="A generated public key for a Redshift connection"/>
 
+#### About the Bastion server in AWS:
+
+A bastion server is really just a host that dbt Cloud can open up an SSH connection to, and given that dbt Cloud is generally only sending queries, and not transmitting large data volumes, you can safely assume that the bastion server can run on a small AWS instance. I'd offer the following recommendations:
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]: suggesting making this into a toggle and summarizing. also is my sug below technically accurate?

>dbt Cloud only sends queries and doesn't transmit large data volumes, which means the bastion server can run on an AWS instance of any size

```suggestion
<details>
  <summary>What is a Bastion server?</summary>
  <div>
    <div>A bastion server in <a href="https://aws.amazon.com/blogs/security/how-to-record-ssh-sessions-established-through-a-bastion-host/">Amazon Web Services (AWS)</a> is a host that allows dbt Cloud to open an SSH connection. <br></br>
    
dbt Cloud only sends queries and doesn't transmit large data volumes. This means the bastion server can run on an AWS instance of any size, like a t2.small instance or t2.micro.<br></br>
    
Make sure the location of the instance is the same Virtual Private Cloud (VPC) as the Redshift instance, and configure the security group for the bastion server to ensure that it's able to connect to the warehouse port.
    </div>
  </div>
</details>
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 2]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-ssh-tunnel.png" title="A generated public key for a Redshift connection"/>
 
+#### About the Bastion server in AWS:
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md:46]:
```suggestion

```

File: website/docs/docs/get-started/connect-your-database.md [hunk 3]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+
+A bastion server is really just a host that dbt Cloud can open up an SSH connection to, and given that dbt Cloud is generally only sending queries, and not transmitting large data volumes, you can safely assume that the bastion server can run on a small AWS instance. I'd offer the following recommendations:
+
+- A t2.small instance is likely sufficiently sized for this purpose (even a t2.micro will probably suffice, in all honesty)
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion

```

File: website/docs/docs/get-started/connect-your-database.md [hunk 4]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+- Make sure the instance is located in the same VPC as the Redshift instance, and configure the security group for the bastion server to ensure that it is able to connect to the warehouse port
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion

```

File: website/docs/docs/get-started/connect-your-database.md [hunk 5]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+
+#### Configuring the Bastion Server in AWS:
+
+When you configure the tunnel in dbt Cloud, you'll need to provide the hostname/IP of your bastion server, as well as a username and port, of your choosing, that dbt Cloud will connect to:
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
To configure the SSH tunnel in dbt Cloud, you'll need to provide the hostname/IP of your bastion server, username, and port, of your choosing, that dbt Cloud will connect to. Review the following steps:
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 6]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+
+- Make sure that the bastion server has its network security rules set up properly to accept connections from the dbt Cloud IP addresses on whatever port you configure for the bastion server in dbt Cloud
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
- Verify the bastion server has its network security rules set up to accept connections from the [dbt Cloud IP addresses](/docs/deploy/regions-ip-addresses) on whatever port you configured.
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 7]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+- You'll need to set up the user account (I'm using the username dbtcloud below) via the bastion servers instance's CLI:
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
- Set up the user account by using the bastion servers instance's CLI, The following example uses the username `dbtcloud:`
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 8]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+    
+    `sudo groupadd dbtcloud`
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]: hey @will-sargent-dbtlabs is the example suppose to look like this screenshot?(2 lines) or is it supposed to take up 6 lines?
Screenshot 2022-12-13 at 14 58 52

File: website/docs/docs/get-started/connect-your-database.md [hunk 9]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+    
+    `mkdir ~/.ssh`
+    
+    `chmod 700 ~/.ssh`
+    
+    `touch ~/.ssh/authorized_keys`
+    
+    `chmod 600 ~/.ssh/authorized_keys`
+    
+- You'll need to copy-paste the generated public key from dbt Cloud into the authorized_keys file
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
- Copy and paste the dbt Cloud generated public key, into the authorized_keys file.
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 10]
```diff
@@ -43,6 +43,38 @@ Once the connection is saved, a public key will be generated and displayed for t
...
+
+With the Bastion EC2 instance in place, and the user properly configured, you should be able to successfully connect to the Redshift environment.
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]: hey @will-sargent-dbtlabs - should the Bastion EC2 instance be referenced earlier in the doc? I'm trying to understand why we're specifying EC2 instance at this stage?

File: website/docs/docs/get-started/connect-your-database.md (hunk 2 above)
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
#### About the Bastion server in AWS
```

**Comment by mirnawong1** [comment]
hey @will-sargent-dbtlabs this is such great content to augment the docs! really appreciate the thoroughness and I've left some suggestions for you to review. let me know your thoughts!

File: website/docs/docs/get-started/connect-your-database.md (hunk 8 above)
Comment (will-sargent-dbtlabs) [reply website/docs/docs/get-started/connect-your-database.md]: 6 line is better @mirnawong1 , thanks!

File: website/docs/docs/get-started/connect-your-database.md (hunk 10 above)
Comment (will-sargent-dbtlabs) [reply website/docs/docs/get-started/connect-your-database.md]: Yeah, this summary line isn't quite there..

Comment (will-sargent-dbtlabs) [reply website/docs/docs/get-started/connect-your-database.md]: @mirnawong1 maybe something like:  "The Bastion server should now be ready for dbt Cloud to use as a tunnel into the Redshift environment."

**Comment by will-sargent-dbtlabs** [comment]
@mirnawong1 Thanks so much for cleaning this up!  This is looking super helpful.  I think you are right in that the last line tweak out to get this thing wrapped up.

File: website/docs/docs/get-started/connect-your-database.md [hunk 11]
```diff
@@ -37,12 +37,52 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
...
+    
+    `chmod 700 ~/.ssh`
+    
+    `touch ~/.ssh/authorized_keys`
+    
+    `chmod 600 ~/.ssh/authorized_keys`
+    
+- Copy and paste the dbt Cloud generated public key, into the authorized_keys file.
+
+With the Bastion EC2 instance in place, and the user properly configured, you should be able to successfully connect to the Redshift environment.
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
The Bastion server should now be ready for dbt Cloud to use as a tunnel into the Redshift environment.
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 12]
```diff
@@ -37,12 +37,52 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
...
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-ssh-tunnel.png" title="A generated public key for a Redshift connection"/>
 
+#### About the Bastion server in AWS
+
+<details>
+  <summary>What is a Bastion server?</summary>
+  <div>
+    <div>A bastion server in <a href="https://aws.amazon.com/blogs/security/how-to-record-ssh-sessions-established-through-a-bastion-host/">Amazon Web Services (AWS)</a> is a host that allows dbt Cloud to open an SSH connection. <br></br>
+    
+dbt Cloud only sends queries and doesn't transmit large data volumes. This means the bastion server can run on an AWS instance of any size, like a t2.small instance or t2.micro.<br></br>
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
dbt Cloud only sends queries and doesn't transmit large data volumes. This means the bastion server can run on an AWS instance of any size, like a t2.small instance or t2.micro.<br></br><br></br>
```

File: website/docs/docs/get-started/connect-your-database.md [hunk 13]
```diff
@@ -37,12 +37,52 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
...
+
+
+#### Configuring the Bastion Server in AWS:
+
+To configure the SSH tunnel in dbt Cloud, you'll need to provide the hostname/IP of your bastion server, username, and port, of your choosing, that dbt Cloud will connect to. Review the following steps:
+
+- Verify the bastion server has its network security rules set up to accept connections from the [dbt Cloud IP addresses](/docs/deploy/regions-ip-addresses) on whatever port you configured.
+- Set up the user account by using the bastion servers instance's CLI, The following example uses the username `dbtcloud:`
+    
+    `sudo groupadd dbtcloud`
```
Comment (mirnawong1) [inline website/docs/docs/get-started/connect-your-database.md]:
```suggestion
    `sudo groupadd dbtcloud`<br></br>
```

**Comment by will-sargent-dbtlabs** [comment]
Latest changes look nice @mirnawong1 !

**Review by mirnawong1 — APPROVED** [review]
looks good @will-sargent-dbtlabs , approved and ready for you to merge! thanks again for opening this up!

**Pull request merged**